common/defines/babylon_defines.txt

```diff
@@ -125,3 +128,13 @@ NPolitics = {
 
 	SECESSION_WEEKLY_BASE_CHANCE = 0.002		# 0.001 weekly base chance for culture to start secession movement
 }
+
+NEconomy = {
+	PRICE_RANGE = 0.99	#0.75					# min price of a good is base * (1 - PRICE_RANGE), Max price is base * (1 + PRICE_RANGE), PRICE_RANGE is locked to [0-1]
+
+	GOODS_SHORTAGE_PENALTY_THRESHOLD = 0.5		# If supply / demand is lower than this, start applying output penalties, by default this should be ( 1 / BUY_SELL_DIFF_AT_MAX_FACTOR )
+	GOODS_SHORTAGE_PENALTY_INCREASE_SPEED = 0.01  	# Goods shortage penalty increases by this fraction of the target each day
+	GOODS_SHORTAGE_PENALTY_DECREASE_SPEED = 0.01  	# Goods shortage penalty goes down by this fraction of GOODS_SHORTAGE_PENALTY_MAX each day
+	GOODS_SHORTAGE_PENALTY_MIN = 0.05			# Goods shortage penalty can't be lower than this if there is any shortage
+	GOODS_SHORTAGE_PENALTY_MAX = 0.75	#0.5	# Goods shortage penalty can't be higher than this
```

Comment:
Huge change for player, may want to inform them.

Reply:
I'll be sure to have it as part of a writeup.